ansible/config_sonic_basedon_testbed.yml

```diff
@@ -174,51 +162,27 @@
         state: directory
         mode: '0755'
       become: true
-
-    - name: Create telemetry server private key
-      openssl_privatekey:
-        path: "{{ server_key }}"
-        size: 2048
-        mode: '0755'
-      become: true
-
-    - name: create telemetry server csr
-      openssl_csr:
-        path: "{{ telemetry_certs['server_csr'] }}"
-        privatekey_path: "{{ server_key }}"
-      become: true
-
-    - name: Generate a Self Signed OpenSSL telemetry server certificate
-      openssl_certificate:
-        path: "{{ server_cer }}"
-        privatekey_path: "{{ server_key }}"
-        csr_path: "{{ server_csr }}"
-        subject:
-          commonName: ndastreamingservertest
-        provider: selfsigned
-      become: true
-
-    - name: Create telemetry dsmsroot private key
-      openssl_privatekey:
-        path: "{{ dsmsroot_key }}"
-        size: 2048
-        mode: '0755'
-      become: true
-
-    - name: create telemetry dsmsroot csr
-      openssl_csr:
-        path: "{{ dsmsroot_csr }}"
-        privatekey_path: "{{ dsmsroot_key }}"
+    
+    - name: Generate server cert using openssl. This need to be copied and renamed as dsmsroot.cer on ptfdocker
```

Comment:
Should the "this needs to be copied..." notes be comments or do you feel that they should be in the output logs?

Reply:
Not needed in output logs.

Reply:
If it's in the task name, it shows up in the logs like you can see here: https://sonic-jenkins.westus2.cloudapp.azure.com/job/common/job/sonic-mgmt-testing/job/sonic-mgmt-pr/196/consoleFull

Reply:
Had offline discussion. added comment instead of the task name